README.md

```diff
@@ -0,0 +1,34 @@
+# EquipmentRentalService
+backend service for my engineers thesis
+
+
+
+## Table of Contents
+* [General Info](#general-information)
+* [Technologies Used](#technologies-used)
+* [Features](#features)
+* [Project Status](#project-status)
+
+
+## General Information
+This project is a microserivce to rental a workout equipment.
+The project contains a android client aplicatoion and backend in SpringBoot framework.
+
+## Technologies Used
+- Android Studio - moblie client application for frontend.
```

Comment:
i believe you use Android Studio for the frontend app but I dont think you do in this Service :)